src/toil/server/app.py

```diff
@@ -12,6 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 import argparse
+from configargparse import ArgumentParser
```

Comment:
Why do we sometimes import `ArgParser` and sometimes import `ArgumentParser`? Can we do one consistently?